tests/e2e/logging/test_openshift-logging.py

```diff
@@ -75,6 +75,7 @@ def teardown(cbp_obj, sc_obj):
 @pytest.mark.usefixtures(
     test_fixture.__name__
 )
+@ignore_leftovers
 class Test_openshift_logging_on_ocs(E2ETest):
```

Comment:
Please change class name to TestOpenshiftLoggingOnOCS

Reply:
Done